GCMessengerSDKSample/Podfile.lock

```diff
@@ -0,0 +1,133 @@
+PODS:
```

Comment:
Sounds like a new file, which didn't exist before. Is it safe to have it added into release?